app/ml.py

```diff
@@ -29,7 +29,7 @@ class CityData(BaseModel):
     longitude: float
     rental_price: float
     crime: str
-    pollution: str
+    air_quality_index: str
```

Comment:
Renaming the key here will change the JSON key that WEB gets and will break their side. We'd need to warn them if you want to do this. You sure it's worth the trouble?